### DISCUSSION TITLE
PVE 8.2.2 LXC 运行 23.05-snapshot Full Cone NAT 无法启动

### DISCUSSION THREAD

**Discussion by Nehcknarf** [immortalwrt/immortalwrt#1324] [Q&A] [answered]
> Thu Jun  6 00:28:33 2024 daemon.notice procd: /etc/rc.d/S19firewall: nft_try_fullcone: cmd /usr/sbin/nft -c 'add table inet fw4-fullcone-test; add chain inet fw4-fullcone-test dstnat { type nat hook prerouting priority -100; policy accept; fullcone; }; add chain inet fw4-fullcone-test srcnat { type nat hook postrouting priority -100; policy accept; fullcone; }; ' 2>/dev/null

> Thu` Jun  6 00:28:33 2024 daemon.notice procd: /etc/rc.d/S19firewall: nft_try_fullcone failed, disable fullcone globally

PVE 8.2.2 Kernel 6.8.4-3-pve
LXC 容器运行 23.05-snapshot 出现 Full Cone NAT 无法启动的状况
是否是因为内核版本太新，导致 kmod-nft-fullcone 无法正确加载？

**Comment by 1715173329** [comment, marked as answer]
你内核没有 fullcone，怎么开？

**Reply by Nehcknarf** [reply]
看来还是KVM跑合适点，感谢天灵大佬解答
